modules/widgets/milestone/style-admin.css

```diff
@@ -0,0 +1,19 @@
+.milestone-widget fieldset {
+	margin-bottom: 1em;
+}
+.milestone-widget .day,
+.milestone-widget .hour,
+.milestone-widget .minutes,
+.milestone-widget .seconds {
+	text-align: right;
+	width: 2em;
+}
+.milestone-widget .year {
+	text-align: right;
+	width: 3.25em;
+}
+.milestone-widget .assistive-text {
```

Comment:
Could we just use WP's native screenreader text classes for this instead of rolling our own?